src/model/qubo.jl

```diff
@@ -42,79 +49,43 @@ MOI.supports_constraint(::QUBOModel{T}, ::Type{VI}, ::Type{MOI.ZeroOne}) where {
 MOI.supports_add_constrained_variable(::QUBOModel{T}, ::Type{MOI.ZeroOne}) where {T} = true
 
 function MOI.add_constraint(::QUBOModel, vi::VI, ::MOI.ZeroOne)
-    MOI.ConstraintIndex{VI,MOI.ZeroOne}(vi.value)
+    return MOI.ConstraintIndex{VI,MOI.ZeroOne}(vi.value)
 end
 
 # -*- :: get + set :: -*- #
 function MOI.get(model::QUBOModel, ::MOI.ObjectiveSense)
-    return model.sense
+    return model.os
```

Comment:
This is a general comment. Using shorter var names is not always better. `sense` is far more informative than `os`, and only 3 characters longer. This can be remedied with in-code documentation. Try using Copilot to help with this (or other language generative models)